en-US/focus-ios.xliff

```diff
@@ -71,6 +71,11 @@
         <target>Help</target>
         <note>Label for row in About screen</note>
       </trans-unit>
+      <trans-unit id="About.rowPrivacy">
+        <source>Privacy Notice</source>
+        <target>Privacy Notice</target>
+        <note>Label for row in About screen</note>
```

Comment:
Nit for future strings: this sounds like a developer comment (I guess rows is what you use to create the layout?)

My feeling is this is a "Link to Privacy Notice in the About screen", or something along that line